external-crates/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -283,6 +302,9 @@ impl Interpreter {
                         .instantiate_generic_function(idx, current_frame.ty_args())
                         .map_err(|e| set_err_info!(current_frame, e))?;
                     let func = resolver.function_from_instantiation(idx);
+                    // Compiled out in release mode
+                    let _func_name = func.pretty_string();
```

Comment:
This and the above feels a bit icky. Could you add the `debug_assertions` macro on these? Or further gate them in some way (e.g., passing the `func` into the below macro.

Reply:
Fixed